spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -1771,6 +1776,7 @@ def _do_load(self, graph_changed):
         executor = self._run_algorithms(
             inputs, algorithms, [], tokens, required_tokens, "loading",
             optional_algorithms)
+        self._no_sync_changes = executor.get_item("NoSyncChanges")
```

Comment:
It isn't clear why this is needed

Reply:
i think this is a slip from having bitfield stuff inside the ear branches. verification might be to chekc if this is in all_algorithms.

Reply:
Removed